### ISSUE TITLE
key features 

### ISSUE THREAD

**Issue by sfugger00** [CIST2100Project/BusinessApplication#15]
I like the idea, however, there really isn't any key features that make this game stand out over other RPGs. I suggest putting more detail into talking about what makes this game unique and talk about features in your games that others don't have

**Comment by genesiscartagena** [comment]
Exactly. You could add a description of what the game is about, or create a description of a few games to attract more customers. Also, you can even let the customers vote which game they would play, and based on the results, create games similarly, but with a different storyline.
